packages/core/src/DelegateMixin.js

```diff
@@ -40,9 +44,7 @@ const DelegateMixinImplementation = superclass =>
     }
 
     connectedCallback() {
-      if (super.connectedCallback) {
```

Comment:
guard not necessary if you pass LitElement as a minimum superclass, which always has this method